### PR TITLE
build: Use OVMF binary released from edk2 repo

### PR DESCRIPTION
**Pull request by rbradford**
Signed-off-by: Rob Bradford <robert.bradford@intel.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -39,12 +39,7 @@ pipeline{
 								installAzureCli()
 							}
 						}
-						stage ('Download assets') {
-							steps {
-								sh "mkdir ${env.HOME}/workloads"
-								sh 'az storage blob download --container-name private-images --file "$HOME/workloads/OVMF-83041af43c.fd" --name OVMF-83041af43c.fd --connection-string "$AZURE_CONNECTION_STRING"'
-							}
-						}
+
```
Comment (sboeuf) [inline Jenkinsfile:42]: Well you can also remove the `Install azure-cli` stage from both `Worker build` and `Worker Build (musl)`

**Review by sboeuf — APPROVED** [review]
Just need to remove a bit more, but otherwise the PR looks good.

One thing about the automation of the EDK2 build. You pushed https://github.com/cloud-hypervisor/edk2/commit/40fbc7e24cc0be9cbc349a911fdd3f451068ad57 to the cloud-hypervisor fork of EDK2, but I was expecting we would directly use the upstream EDK2 master branch. But I also understand we couldn't automate the build in that case.
So I guess we're gonna push the lastest EDK2 master to cloud-hypervisor fork from time to time in order to rely on the latest source from EDK2?

**Review by sboeuf — APPROVED** [review]

**Comment by rbradford** [comment]
> Just need to remove a bit more, but otherwise the PR looks good.
> 
> One thing about the automation of the EDK2 build. You pushed [cloud-hypervisor/edk2@40fbc7e](https://github.com/cloud-hypervisor/edk2/commit/40fbc7e24cc0be9cbc349a911fdd3f451068ad57) to the cloud-hypervisor fork of EDK2, but I was expecting we would directly use the upstream EDK2 master branch. But I also understand we couldn't automate the build in that case. So I guess we're gonna push the lastest EDK2 master to cloud-hypervisor fork from time to time in order to rely on the latest source from EDK2?

I can't think how you would do the automation using the upstream branch (and surely you'd only want the latest released tag). Hopefully at some point in the future all the patches will be integrated and there might be a released binary published (although AFAICT there is no released binary from upstream.)

**Comment by sboeuf** [comment]
> > Just need to remove a bit more, but otherwise the PR looks good.
> > One thing about the automation of the EDK2 build. You pushed [cloud-hypervisor/edk2@40fbc7e](https://github.com/cloud-hypervisor/edk2/commit/40fbc7e24cc0be9cbc349a911fdd3f451068ad57) to the cloud-hypervisor fork of EDK2, but I was expecting we would directly use the upstream EDK2 master branch. But I also understand we couldn't automate the build in that case. So I guess we're gonna push the lastest EDK2 master to cloud-hypervisor fork from time to time in order to rely on the latest source from EDK2?
> 
> I can't think how you would do the automation using the upstream branch (and surely you'd only want the latest released tag). Hopefully at some point in the future all the patches will be integrated and there might be a released binary published (although AFAICT there is no released binary from upstream.)

Yes I thought the same when I saw your code. No big deal, we'll rebase your automation patch from time to time and push to cloud-hypervisor/edk2 ch in order to generate a new binary.

**Pull request merged**